### PR TITLE
Add metrics for each Neo4j writer to identify perf bottleneck

### PR DESCRIPTION
**Pull request by Dalamar42**

### CODE REVIEW

File: atlas-api/src/main/java/org/atlasapi/AtlasPersistenceModule.java [hunk 1]
```diff
@@ -633,6 +633,6 @@ public Neo4jModule neo4jModule() {
 
     @Bean
     public Neo4jContentStore neo4jContentStore() {
-        return neo4jModule().neo4jContentStore();
+        return neo4jModule().neo4jContentStore(metricsModule.metrics());
```
Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/AtlasPersistenceModule.java]: Why not just wire the metrics module into neo4j module and just return a ready content store from there?

Comment (Dalamar42) [inline atlas-api/src/main/java/org/atlasapi/AtlasPersistenceModule.java]: The neo4j module is not an `@Configuration` class. I am continuing the pattern set by the ES module where the ES module itself is just a vanilla Java class and the atlas persistence module is the first Spring component in the wiring chain.

Comment (emilssolmanis) [inline atlas-api/src/main/java/org/atlasapi/AtlasPersistenceModule.java]: Meh. Fine.

**Comment by emilssolmanis** [comment]
lgtm

**Pull request merged**